examples/manage-dns/init.pp

```diff
@@ -0,0 +1,28 @@
+route53_zone { 'puppetlabs.com.':
+  ensure => present,
+}
+
+route53_a_record { 'local.puppetlabs.com.':
+  ensure => present,
+  zone   => 'puppetlabs.com.',
+  ttl    => 3000,
+  values => ['127.0.0.1'],
+}
+
+route53_ns_record { 'local.puppetlabs.com.':
+  ensure => present,
+  zone   => 'puppetlabs.com.',
+  ttl    => 17200,
+  values => [
+    'ns-377.awsdns-47.com.',
+    'ns-1724.awsdns-23.co.uk.',
+    'ns-969.awsdns-57.net.',
+  ]
+}
+
+route53_txt_record { 'local.puppetlabs.com.':
+  ensure => present,
+  zone   => 'puppetlabs.com.',
+  ttl    => 17200,
+  values => '"message"'
```

Comment:
I understand that the double quotes are needed when entering it in the console, and this is consistent with that, but from the puppet side it's not really clear.